README.md

```diff
@@ -154,3 +154,16 @@ found in [the Buildbarn deployments repository](https://github.com/buildbarn/bb-
 There is a [#buildbarn channel on buildteamworld.slack.com](https://bit.ly/2SG1amT)
 that you can join to get in touch with other people who use and hack on
 Buildbarn.
+
+# Commercial Support
+
+Buildbarn has an active and enthusiastic community. Though we try to help and
+support those who have issues or questions, sometimes organisations need more
+dedicated support. Here is a list of community members who you can contact if
+you require commercial support.
+
+[Finn Ball](mailto:finn.ball@codificasolutions.com) - Freelance Consultant
```

Comment:
Great! Could we also mention the following two things?

- That people can submit PRs if they want to have their name / company added to this list.
- That having your name listed there isn't necessarily an endorsement.

Apart from that: looks perfect!